app/src/main/res/values/strings.xml

```diff
@@ -897,20 +897,20 @@
     <string name="reverse">Reverse</string>
     <string name="list_options">List options</string>
     <string name="ssaid">SSAID</string>
-    <string name="ssaid_info">SSAID (<tt>Settings.Secure.ANDROID_ID</tt>) is a device identifier assigned to each app (from Android Oreo). It is widely used by apps to track users.</string>
+    <string name="ssaid_info">SSAID (<tt>Settings.Secure.ANDROID_ID</tt>) is a device identifier assigned to each app (from Android 8 \"Oreo\" onwards). It is widely used by apps to track users.</string>
     <string name="copied_to_clipboard">Copied to clipboard.</string>
     <string name="failed_to_change_ssaid">Could not change SSAID</string>
-    <string name="restart_to_reflect_changes">Please restart immediately to reflect changes</string>
+    <string name="restart_to_reflect_changes">Restart the device immediately so that the new changes work.</string>
```

Comment:
_might_ work. This is, unfortunately, not guaranteed to work i.e. it works most of the time but may not always work due to OS-specific issues.